gui/slick/interfaces/default/home.mako

```diff
@@ -222,7 +224,7 @@ $(document).ready(function(){
             4: { sorter: 'quality' },
             5: { sorter: 'eps' },
             % if sickbeard.FILTER_ROW:
-            6: { filter : 'parsed' }
+                7: { filter : 'parsed' }
             % endif
         },
```

Comment:
I didn't know we could change the order of the column...
But if you look a couple of lines above my change, you can see that column 3 is missing too.
Should just add `6: {}` as I don't need anything special?